### PR TITLE
[SiVal] Test plan update for adc_ctrl

### PR DESCRIPTION
**Pull request by jdonjdon**
1. Added list of features to the adc_ctrl specification.
2. Moved adc_ctrl chip tests to chip_adc_ctrl_testplan.hjson
3. Added si_stage: SV* test cases to cover functionality enumerated in the list of features.
4. Updated the pre-silicon test cases with relevant SV* labels.

### CODE REVIEW

File: hw/top_earlgrey/data/chip_testplan.hjson [hunk 1]
```diff
@@ -1340,7 +1341,11 @@
       desc: '''Verify that the USB stops responding when its clock is disabled in active state.
 
             Configure the pwrmgr CONTROL CSR with USB_CLK_EN_ACTIVE off, and issue a CSR read
+<<<<<<< HEAD
```
Comment (timothytrippel) [inline hw/top_earlgrey/data/chip_testplan.hjson:1344]: looks like you have a merge conflict that you need to resolve here?

File: hw/ip/adc_ctrl/data/adc_ctrl.hjson [hunk 2]
```diff
@@ -80,6 +80,26 @@
       desc: "End-to-end bus integrity scheme."
     }
   ]
+  features: [
+    {
+      name: "ADC_CTRL.MODE.NORMAL"
+      desc: '''The ADC will always be enabled.
```
Comment (timothytrippel) [inline hw/ip/adc_ctrl/data/adc_ctrl.hjson:86]:
```suggestion
      desc: '''In this mode, the ADC will always be enabled and consuming power.
```

Comment (jdonjdon) [reply hw/ip/adc_ctrl/data/adc_ctrl.hjson:86]: fixed

**Review by timothytrippel — APPROVED** [review]

**Comment by timothytrippel** [comment]
CHANGE AUTHORIZED: hw/top_earlgrey/data/chip_testplan.hjson
CHANGE AUTHORIZED: hw/top_earlgrey/data/chip_adc_ctrl_testplan.hjson

**Comment by jdonjdon** [comment]
CHANGE AUTHORIZED: hw/top_earlgrey/data/chip_testplan.hjson
CHANGE AUTHORIZED: hw/top_earlgrey/data/chip_adc_ctrl_testplan.hjson

**Comment by timothytrippel** [comment]
CHANGE AUTHORIZED:hw/ip/adc_ctrl/data/adc_ctrl.hjson

**Comment by timothytrippel** [comment]
CHANGE AUTHORIZED: hw/ip/adc_ctrl/data/adc_ctrl.hjson

**Pull request merged**